api/RSSFeed.php

```diff
@@ -240,6 +240,22 @@ public function setTemplate($template) {
 	public function getTemplate() {
 		return $this->template;
 	}
+
+	/**
+	 * Returs the ordered list of preferred templates for rendering this form
```

Comment:
The PHPDoc is misleading, it refers to `Form` in the `RSSFeed` class (and misspelled "Returns")